2-Browsers/Week1/homework/ex5-catWalk/index.js

```diff
@@ -15,8 +15,35 @@
    for 5 seconds, and then replace the img with the original image and have it 
    continue the walk.
 -----------------------------------------------------------------------------*/
+
+const catImage = document.querySelector('img');
+catImage.style.left = '0px';
+const catImageWidth = catImage.width;
+const endOfScreen = window.innerWidth - catImageWidth;
+const midOfScreen = endOfScreen / 2;
+let catPosition = 0;
+let dancing = false;
+
 function catWalk() {
-  // TODO complete this function
-}
+  const startWalking = setInterval(function () {
```

Comment:
To fully match the assignment you should call `catWalk()` from the `setInterval`, not another function. You'll need to move the `setInterval` call outside of `catWalk()` to do that.